temporal-sdk/src/main/java/io/temporal/internal/nexus/NexusTaskHandlerImpl.java

```diff
@@ -208,15 +214,27 @@ private void convertKnownFailures(Throwable failure) {
   }
 
   private StartOperationResponse handleStartOperation(
-      OperationContext.Builder ctx, StartOperationRequest task)
-      throws InvalidProtocolBufferException {
+      OperationContext.Builder ctx, StartOperationRequest task) {
     ctx.setService(task.getService()).setOperation(task.getOperation());
 
     OperationStartDetails.Builder operationStartDetails =
         OperationStartDetails.newBuilder()
             .setCallbackUrl(task.getCallback())
             .setRequestId(task.getRequestId());
     task.getCallbackHeaderMap().forEach(operationStartDetails::putCallbackHeader);
+    task.getLinksList()
```

Comment:
@bergundy is it fair to say their is no way to set this right now?

Reply:
Are you referring to setting link from the caller workflow? If so, then yes, the server will set this automatically without an ability to override.